### DISCUSSION TITLE
Integrate autocomplete with fig

### DISCUSSION THREAD

**Discussion by FraWolf** [foundry-rs/foundry#2729] [Ideas]
Hello,
Im starting using foundry in my Solidity journey and I was thinking about if you could integrate the autocompletition with [Fig](https://fig.io/).
Thanks!

**Comment by onbjerg** [comment]
Probably not something we will maintain ourselves here. It's possible for anyone to write bindings for any tool, so anyone is welcome to do that :)

**Comment by Genysys** [comment]
I have this almost done and I am happy to publish it if there is interest.
Lost motivation when I realised that foundry has bash auto complete already
